go/runtime/bundle/manager.go

```diff
@@ -236,20 +247,44 @@ func (m *Manager) Queue(runtimeID common.Namespace, manifestHashes []hash.Hash)
 	}
 	m.downloadQueue[runtimeID] = hashes
 
-	// Trigger immediate download of new bundles.
+	// Trigger immediate download and clean-up of stale bundles.
 	select {
-	case m.downloadCh <- struct{}{}:
+	case m.triggerCh <- struct{}{}:
 	default:
 	}
 }
 
-// Download tries to download bundles in the queue.
-func (m *Manager) Download() {
+// Cleanup initiates the clean-up of regular bundles for the specified runtime,
```

Comment:
```suggestion
// Cleanup initiates the clean-up of bundles for the specified runtime,
```
Here you are mentioning regular, lower stale. Confusing.